crates/jit-icache-coherence/src/libc.rs

```diff
@@ -60,10 +60,53 @@ pub(crate) fn pipeline_flush_mt() -> Result<()> {
         // In any other case we got an actual error, so lets propagate that up
         e => e?,
     }
+    #[cfg(all(target_arch = "riscv64", target_os = "linux"))]
+    return riscv_flush_icache(0, 0);
```

Comment:
I don't think we need to do this here. With the way the API for this crate is set up, we should always have a `clear_cache` operation preceding this. So we are effectively doing the cache flushing twice!

Reply:
ok,thanks.